python/mxnet/test_utils.py

```diff
@@ -1417,6 +1417,34 @@ def download(url, fname=None, dirname=None, overwrite=False):
     logging.info("downloaded %s into %s successfully", url, fname)
     return fname
 
+def get_images(directory):
```

Comment:
This is not reusable even though you moved the function in test_utils

Reply:
@zhreshold can you elaborate?

Reply:
The urls and operations are hard-coded, which means it's a one-time util function. Besides, I don't find it's reused anywhere else in the repo, am I missing something?

Reply:
I don't see the problem, they are the demo images. What do you suggest?

Reply:
Functions included in test_utils should still be something users/developers could reuse because it's in package, not in tests. This is however purely a util script for grabbing test images, so I suggest put it back in unittest or keep it but made flexible with arguments for example:
```
def get_images(directory, url, compression='tar')
```

Reply:
Other things LGTM, and can be merged once resolved.